packages/lib/podspec/todo.md

```diff
@@ -0,0 +1,5 @@
+# todo
+
+- [ ] Fully comprehensive tests
+- [ ] Merge/extend entries
```

Comment:
what's an example of this?

Reply:
This would be where I define a set of entries a POD should have, then later extend it by adding more entries. It's directly equivalent to the `.extend()` method on ZodObject types.